docs/api/checkouts.rst

```diff
@@ -1,235 +1,560 @@
+| shipping_address[country]    | request        | Id of the country                                                                                    |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| shipping_address[province]   | request        | *(optional)* Id of the province                                                                      |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[first_name]  | request        | *(optional)* Firstname for billing address                                                           |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[last_name]   | request        | *(optional)* Lastname for billing address                                                            |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[city]        | request        | *(optional)* City name                                                                               |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[postcode]    | request        | *(optional)* Postcode                                                                                |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[street]      | request        | *(optional)* Address line 1                                                                          |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[country]     | request        | *(optional)* Id of the country                                                                       |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+| billing_address[province]    | request        | *(optional)* Id of the province                                                                      |
++------------------------------+----------------+------------------------------------------------------------------------------------------------------+
+
+.. note::
+
+    Remember a cart with `id = 21` from the cart docs? We will take the same cart as a exemplary cart for checkout process
```

Comment:
A link to the Cart API documentation needed here (:doc:`from the Cart API documentation <link>`).